toolkits/x/evals/eval_x_tools.py

```diff
@@ -78,9 +77,9 @@ def x_eval_suite() -> EvalSuite:
             "at Arcade AI!'"
         ),
         expected_tool_calls=[
-            ExpectedToolCall(
-                func=post_tweet,
-                args={
+            (
```

Comment:
We are in the process of moving to the `ExpectedToolCall(...)` syntax. This can be reverted back